### PR TITLE
Add Kamereon client

### PR DESCRIPTION
**Pull request by epenet**

### CODE REVIEW

File: src/renault_api/kamereon.py [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+        endpoint: str,
+        params: Optional[Dict[str, str]] = None,
+        schema: Optional[Schema] = None,
+    ) -> model.KamereonVehicleDataResponse:
+        """GET to /v{endpoint_version}/cars/{vin}/{endpoint}."""
+        path_to_car = self._get_path_to_car(account_id, endpoint_version, vin)
+        return cast(
+            model.KamereonVehicleDataResponse,
+            await self._request(
+                schema=schema or model.KamereonVehicleDataResponseSchema,
```
Comment (Quentame) [inline src/renault_api/kamereon.py:160]: Why having `schema or` here ?
`schema` param is actually never set.

File: src/renault_api/kamereon.py [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+        endpoint: str,
+        data: Dict[str, Any],
+        schema: Optional[Schema] = None,
+    ) -> model.KamereonVehicleDataResponse:
+        """POST to /v{endpoint_version}/cars/{vin}/actions/{endpoint}."""
+        path_to_car = self._get_path_to_car(account_id, endpoint_version, vin)
+        return cast(
+            model.KamereonVehicleDataResponse,
+            await self._request(
+                schema=schema or model.KamereonVehicleDataResponseSchema,
```
Comment (Quentame) [inline src/renault_api/kamereon.py:253]: > Why having `schema or` here ?
> `schema` param is actually never set.

Same here

**Review by Quentame — APPROVED** [review]
Some comments, otherwise LGMT

**Comment by epenet** [comment]
@Quentame it was planned for future use (when the ResponseData shemas are expanded)
I think it's ok to remove for now, and add again later.

**Comment by Quentame** [comment]
Or then 👌

**Pull request merged**